website/src/lib/blog.ts

```
@@ -0,0 +1,48 @@
+
+/**
+ * Read a post's date out of its filename, e.g.
+ * `src/content/blog/2024-08-07-version-1.md`. Posts carry no `date` in
+ * frontmatter, so the filename is the only source.
+ *
+ * @param post - The blog post.
+ * @returns The post's publication date.
+ */
+export function getPostDate(post: Post): Date {
+  // Collection entries only have a `filePath` when their loader is file-backed.
+  // The blog always is, so an absent path means something is badly wrong.
```

ESLint

[error] Expected a block comment instead of consecutive line comments.

(multiline-comment-style)